### PR TITLE
Added auto focus prop for Modal functions.

### PR DESCRIPTION
**Pull request by spideeee**
#11052

### CODE REVIEW

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/tree/feature-3.7.0

File: components/modal/Modal.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ export interface ModalFuncProps {
   style?: React.CSSProperties;
   type?: string;
   keyboard?: boolean;
+  autoFocusOk?: boolean;
```
Comment (yesmeck) [inline components/modal/Modal.tsx]: Could you add it to documentation?

Comment (yesmeck) [inline components/modal/Modal.tsx]: Could you add it to documentation?

**Review by yesmeck — CHANGES_REQUESTED** [review]

File: components/modal/Modal.tsx (hunk 1 above)
Comment (spideeee) [reply components/modal/Modal.tsx]: Done, feel free to update the description.

File: components/modal/index.en-US.md [hunk 2]
```diff
@@ -72,6 +72,7 @@ The properties of the object are follows:
 | zIndex | The `z-index` of the Modal | Number | 1000 |
 | onCancel | Specify a function that will be called when the user clicks the Cancel button. The parameter of this function is a function whose execution should include closing the dialog. You can also just return a promise and when the promise is resolved, the modal dialog will also be closed | function | - |
 | onOk | Specify a function that will be called when the user clicks the OK button. The parameter of this function is a function whose execution should include closing the dialog. You can also just return a promise and when the promise is resolved, the modal dialog will also be closed | function | - |
+| autoFocusOk | To autofocus the OK button, when the modal opens | Boolean | false |
```
Comment (yesmeck) [inline components/modal/index.en-US.md]: We should keep the original behavior, so the default value should be `true`.

**Comment by cheshireoctopus** [comment]
@spideeee - thank you for this!

**Comment by afc163** [comment]
trace #11756

**Pull request closed without merging**